jwt.go

```diff
@@ -101,6 +112,34 @@ func jwtMiddleware() gin.HandlerFunc {
 				return
 			}
 
+			keyParam := getRequestParam(c, "key")
+			bucketParam := getRequestParam(c, "bucket")
+
+			if keyParam == "" {
+				keyParam = getRequestParam(c, "prefix")
+			}
+
+			// Validate JWT bucket and key for all calls except ListBuckets (GET on "/" without bucket or key defined)
+			if (c.Request.Method != "GET" && c.Request.Method != "HEAD") || bucketParam != "" || keyParam != "" {
```

Comment:
For the ListBuckets case, the bucket and key params will already be `""` so the first statement is not needed.

To make it simpler to read, you can remove the top level if and just do
```golang
if keyParam != "" && strings.TrimSpace(claims.Prefix) != "" && !strings.HasPrefix(keyParam, strings.TrimSpace(claims.Prefix)) {
	log.Debugf("JWT token prefix does not match URL object (prefix %s != object %s)", claims.Prefix, strings.TrimSpace(keyParam))
	jwtRequestsMetric.WithLabelValues("false", "JWT token prefix does not match URL object").Inc()
	c.JSON(401, gin.H{"error": "JWT token prefix does not match URL object"})
	c.Abort()
	return
}

if bucketParam != "" && strings.TrimSpace(claims.Bucket) != "" && strings.TrimSpace(claims.Bucket) != bucketParam {
	log.Debugf("JWT token bucket does not match URL bucket (jwt bucket %s != URL bucket %s", claims.Bucket, strings.TrimSpace(bucketParam))
	jwtRequestsMetric.WithLabelValues("false", "JWT token bucket does not match URL bucket").Inc()
	c.JSON(401, gin.H{"error": "JWT token bucket does not match URL bucket"})
	c.Abort()
	return
}
```